src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +83,33 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef CORRADE_TARGET_APPLE
+        Mn::GL::BufferImage2D image = tgt.getDepthTexture().image(
+	            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float}, Mn::GL::BufferUsage::StaticRead);
```

Comment:
To save a bit more perf, you could keep the buffer image around instead of recreating and reallocating it every frame (so using the `image()` variant that takes a reference, not this one that returns it by-value).

~~But to actually make that work, you'd have to `img.release()` the buffer back to this depth format~~ wow, what a dumpster fire of an idea, disregard

So, make `image` a member to avoid the creation/allocation every frame, and then instead of the `release()` below do

```cpp
                Mn::GL::BufferImage2D imageAsRed{ // better name?
                    image.storage(), Mn::PixelFormat::R32F, image.size(), Mn::GL::Buffer::wrap(image.buffer().id(), Mn::ObjectFlag::Created), image.dataSize()};
```

... which basically creates a "buffer view" and should avoid all this verbose complicated ownership transfer.